tests/test_settings.py

```diff
@@ -41,3 +60,60 @@ def test_get_var(self):
             'os.environ', {'FOO': 'notbar'}, clear=True
         ):
             self.assertEqual(get_var('FOO', 'lemon'), 'notbar')
+
+    def test_cas_settings(self,):
+        """Verify CAS settings get set correctly"""
+        with mock.patch.dict('os.environ', {
+            'LORE_USE_CAS': 'yep',
+            'LORE_CAS_URL': 'foobar'
+        }, clear=True):
+            settings_vars = self.reload_settings()
+            self.assertTrue(
+                set(dict(
+                    CAS_ENABLED=True,
+                    LOGIN_URL='/login/',
+                    LOGIN_REDIRECT_URL='/',
+                    CAS_SERVER_URL='foobar'
+                )).issubset(set(settings_vars))
```

Comment:
That is because they took away my beloved (but ambiguous)[assertDictContainsSubset](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertDictContainsSubset), but thanks